### ISSUE TITLE
Server side (Node) logout method call does not work 

### ISSUE THREAD

**Issue by shiftlabs1** [authorizerdev/authorizer-js#32]
**Version:** x.y.z
latest
**Describe the bug**

 Calling logout in node fails with the error 
```
[ { message: 'http: named cookie not present', path: [ 'logout' ] } ]

```

**Steps To Reproduce**
I used both the access_token and the Id token as bearer token and got the same result  
```
 const logoutResp: ApiResponse<GenericResponse> = await authClient.logout({
        Authorization: `Bearer ${accessToken}`
    })
```

**Expected behavior**
access and id tokens should be cleared out from the Db , and response received stating logged out successfully

**Comment by shiftlabs1** [comment]
updateProfile also does not work from node

**Comment by lakhansamani** [comment]
For updateProfile, you need to send `Authorization: Bearer ACCESS_TOKEN` header.

**Comment by shiftlabs1** [comment]
Some function description in the documentation just say SOME ACCESS TOKEN which is confusing 

```
const res = await authRef.deactivateAccount({
  Authorization: `Bearer some_token`,
})
```

**Comment by shiftlabs1** [comment]
@lakhansamani  so access_token works for updateProfile.

**Comment by lakhansamani** [comment]
@shiftlabs1 this is fixed in 1.3.5
Pls update server after 15-20 min https://github.com/authorizerdev/authorizer/actions/runs/7212235569
